docs/src/main/sphinx/release/release-406.md

```diff
@@ -0,0 +1,96 @@
+# Release 406 (18 Jan 2023)
+
+## General
+
+* Add support for [exchange spooling on HDFS](fte-exchange-hdfs) when
+  fault-tolerant execution is enabled. ({issue}`15160`)
+* Improve planner estimates for queries containing outer joins over a subquery
+  involving `ORDER BY` and `LIMIT`. ({issue}`15428`)
+* Improve accuracy memory usage reporting for table scans. ({issue}`15711`)
+* Improve performance of queries parsing date values in ISO 8861 format. ({issue}`15548`)
+* Remove `legacy-phased` execution scheduler. ({issue}`15657`)
```

Comment:
Do mention what configuration needs to be updated if someone was using it for some reason.